files/content/instruction_booklet/booklet_entity/booklet.lua

```diff
@@ -3,13 +3,16 @@ local ui = dofile_once("mods/noita.fairmod/files/content/instruction_booklet/gui
 
 function enabled_changed(entity_id, is_enabled)
 	if is_enabled then
+		GameAddFlagRun("fairmod_dialog_interacting")
 		ui.gui = GuiCreate()
 	else
+		GameRemoveFlagRun("fairmod_dialog_interacting")
 		GuiDestroy(ui.gui)
 	end
 end
 
 function wake_up_waiting_threads()
+	GameHasFlagRun("fairmod_interacted_with_anything_this_frame")
```

Comment:
?

Reply:
uhhh just added that to make the booklet block interactions while open

Reply:
it's checking if the flag exists but doesn't do anything with it

Reply:
> it's checking if the flag exists but doesn't do anything with it

OH lmao that was supposed to be an add flag not has flag